test/e2e/settings/colWidths.spec.js

```diff
@@ -213,6 +218,7 @@ describe('settings', () => {
                 if (index === 0) {
                   return '123';
                 }
```

Comment:
Interesting 🤔 does not always work correctly?

Reply:
It works :) the whole code block looks like
```js
width(index) {
  if (index === 0) {
    return '123';
  }

  return '50';
}
```

Reply:
So it is a code block rule? I've noticed that `return '123';` does not have a blank line. But on the other hand, this would be too verbose. I like it this way so this is only an observation